### PR TITLE
Fix Phoenix.LiveViewTest.page_title/1 to allow for <title> within <svg>

### PR DESCRIPTION
**Pull request by PaulOstazeski**
From [hexdocs.pm/phoenix_live_view/live-layouts.html#updating-document-title](https://hexdocs.pm/phoenix_live_view/live-layouts.html#updating-document-title)

> Because the root layout from the Plug pipeline is rendered outside of LiveView, the contents cannot be dynamically changed. The one exception is the <title> of the HTML document. Phoenix LiveView special cases the @page_title assign to allow dynamically updating the title of the page, which is useful when using live navigation, or annotating the browser tab with a notification.

The current implementation of `Phoenix.LiveViewTest.page_title/1` searches only for a single <title> tag.

The assumption that only one `<title>` tag exists on the page does not hold if the page also includes a title tag within an `<svg>` element. The `page_title/1` function returns `nil` but the HTML for the page title tag contains the expected value.

`<svg><title>Brief, accessible title</title>...</svg> ` is a valid pattern. https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title

Updating the CSS selector from `title` to `head > title` seems to match the [mozilla documentation for the document title tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/title).

Superficially similar to #1496

### CODE REVIEW

File: test/support/live_views/elements.ex [hunk 1]
```diff
@@ -153,6 +153,9 @@ defmodule Phoenix.LiveViewTest.ElementsLive do
     <form id="trigger-form-value" action="/not_found" method="POST" phx-submit="form-submit-trigger"
           phx-trigger-action={@trigger_action}>
     </form>
+
+    <%# @page_title assign is unique %>
+    <svg><title>SVG with title</title></svg>
```
Comment (PaulOstazeski) [inline test/support/live_views/elements.ex:158]: This addition breaks [an existing test for `page_title/1`](https://github.com/phoenixframework/phoenix_live_view/blob/a5fec1d110de8638d1a5042df33e66429b7e2b7a/test/phoenix_live_view/integrations/layout_test.exs#L77). This seems to like such an edge case as to not justify the overhead of a distinct test case.

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:
